pkg/storage/unified/apistore/prepare.go

```diff
@@ -75,7 +75,7 @@ func (s *Storage) prepareObjectForStorage(ctx context.Context, newObject runtime
 		return nil, err
 	}
 	obj.SetRepositoryInfo(repo)
-	obj.SetUpdatedBy("")
+	obj.SetUpdatedBy(user.GetUID())
```

Comment:
Check whether the old behavior is preferred. This change will affect all resources, not just folders.

[PR](https://github.com/grafana/grafana/pull/90513/files#diff-a6d40da7a17d43622287c2147606ef055a394f2110ac283dea3c10d5fe64f216R39) where updatedBy was being set to empty string.